src/AutoRest.CSharp/LowLevel/Output/DpgOutputLibraryBuilder.cs

```diff
@@ -130,7 +130,8 @@ private void CreateModels(IDictionary<InputModelType, ModelTypeProvider> models,
                         actualBase,
                         Array.Empty<InputModelType>(),
                         "Unknown", //TODO: do we need to support extensible enum / int values?
-                        null)
+                        null,
+                        false)
```

Comment:
What's the unknowDeriveModel for? Will we generate this model? If we set it isConfident = false, we will not generate it.

Reply:
should be true. I will fix this.